### ISSUE TITLE
Question: Can python-inpector handle manifest files other than requirements.txt and setup.py?

### ISSUE THREAD

**Issue by shi9qiu** [nexB/python-inspector#153]
Hi,

I am trying to use this tool to generate SBOM files for Python projects. The README says this tool can "parse various manifests and packages files such as pip requirement files, Pipfile, pyproject.toml, poetry.lock, setup.py and setup.cfg and legacy and current metadata file formats for eggs, wheels and sdist". But I find it seems that only requirements.txt and setup.py can be set as the target manifest file.

I wonder if I didn't understand how to use it. Or do you have a plan to develop the function for another manifest file such as "pyproject.toml" or "setup.cfg"?

Thank you!

**Comment by pombredanne** [comment]
Actually this is both true and not true.
The code is https://github.com/nexB/python-inspector/blob/main/src/_packagedcode/pypi.py is able to process all these formats , but only requirements.txt and setup.py have been "wired" in to be used as an input at this stage. The other are in the plan: we have the parsers alright, so the work to get these as inputs should be not too big.

Let me update the description that did mislead you.

**Comment by shi9qiu** [comment]
Thank you very much for the response. Looking forward to the function to process all these formats!

**Comment by pombredanne** [comment]
Let's keep this open so we can track the addition here!
Note also that if you know some Python, help would be much appreciated and always welcomed!

**Comment by shi9qiu** [comment]
Yes, of course! I would like to help with this functional improvement!
